core/src/main/java/org/elasticsearch/action/get/MultiGetResponse.java

```diff
@@ -78,7 +81,7 @@ public String getId() {
          * The failure message.
          */
         public String getMessage() {
-            return this.message;
+            return throwable != null ? throwable.getMessage() : null;
```

Comment:
Would it be beneficial here to return an empty string instead of null? If not, maybe just annotate this with `@Nullable`